### PR TITLE
Add gwf to scs scripts

### PR DESCRIPTION
**Pull request by Vikort**

### CODE REVIEW

File: scripts/kb_scripts/scs_writer.py [hunk 1]
```diff
@@ -0,0 +1,427 @@
...
+  def correct_idtf(self, el):
+    idtf = el["idtf"]
+    is_var = self.is_variable(el["type"])
+    if idtf is None or len(idtf) == 0:
+      if is_var:
+        el["idtf"] = ".._el_{}".format(el["id"].replace("-", "_"))
+      else:
+        el["idtf"] = "..el_{}".format(el["id"].replace("-", "_"))
+    elif is_var:
+      if not idtf.startswith("_"):
```
Comment (ValikVityaz) [inline scripts/kb_scripts/scs_writer.py]: why isn't the same check/correction done for a constant node with an identifier starting with '_'?

File: scripts/kb_scripts/scs_writer.py [hunk 2]
```diff
@@ -0,0 +1,427 @@
...
+        edge_type = e["type"]
+        is_unsupported = False
+        try:
+          symbol = EdgeTypes[edge_type]
+        except KeyError:
+          try:
+            scs_edge_type = UnsupportedEdgeTypes[edge_type]
+            is_unsupported = True
+          except KeyError:
+            msg = "Edge type `{}` is not supported. Please add it into EdgeTypes or into UnsupportedEdgeTypes".format(edge_type)
```
Comment (ValikVityaz) [inline scripts/kb_scripts/scs_writer.py]:
```suggestion
            msg = "Edge type `{}` is unknown. Please add it into EdgeTypes or into UnsupportedEdgeTypes".format(edge_type)
```

File: scripts/kb_scripts/scs_writer.py [hunk 3]
```diff
@@ -0,0 +1,427 @@
...
+        
+        processed_edge = True
+        self.written_elements.append(e["id"])
+        e["idtf"] = alias
+
+    if len(edges_queue) > 0:
+      edge_ids = ""
+      for e in edges_queue:
+        edge_ids += "{} ".format(e["id"])
+      self.log_error("Wasn't able to create edges: {}".format(edge_ids))
```
Comment (ValikVityaz) [inline scripts/kb_scripts/scs_writer.py]: It would be better to specify exactly which edges couldn't be processed.

**Review by ValikVityaz — APPROVED** [review]

**Review by Zioba — APPROVED** [review]

File: scripts/kb_scripts/gwf_to_scs.py [hunk 4]
```diff
@@ -0,0 +1,79 @@
+import argparse
+import os
+import shutil
+import sys
+
+from termcolor import colored
+from tqdm import tqdm
+
+from gwf_parser import GWFParser
```
Comment (KovalM) [inline scripts/kb_scripts/gwf_to_scs.py]: Move GWF parser and SCsWriter to separate folder in kb_scripts as support scripts. In kb_scripts should be only main scripts for prepare KB

File: scripts/kb_scripts/prepare_kb.py [hunk 5]
```diff
@@ -33,6 +34,7 @@ def main(copy_kb_path: str):
                     prepared_repo_path.write(
                         os.path.join(os.path.split(copy_kb_path)[1], path.replace('../', '')) + '\n'
                     )
+            prepared_repo_path.write(os.path.join(os.path.split(copy_kb_path)[1], 'converted'))
```
Comment (KovalM) [inline scripts/kb_scripts/prepare_kb.py]: We already have prepared_kb filder in the root of ostis-web-platform. Why we need converted folder?

**Review by KovalM — CHANGES_REQUESTED** [review]

**Review by KovalM — APPROVED** [review]

**Pull request merged**